src/asyncopenstackclient/client.py

```diff
@@ -22,7 +22,10 @@ def __init__(self, api_name, resources, session=None, api_url=None):
         await self.get_credentials()
         self.api = API(
             api_root_url=self.api_url,
-            headers={'X-Auth-Token': self.session.token},
+            headers={'X-Auth-Token': self.session.token,
```

Comment:
Wouldn't it be nice if it was `pep8` compliant? ;)

Reply:
flake does not complain... it's ok according to pep8